# VDH Data Preprocessing

#### Evan Mitchell (etm8fs)
#### Last update: 05/24/2022

In [2]:
import pandas as pd
import os

### Load data

In [5]:
data_folder = '../data/'
vdh_filename = 'VDH.csv'
vdh_path = os.path.join(data_folder, vdh_filename)
vdh = pd.read_csv(vdh_path)

In [7]:
vdh.head()

Year of Death  Locality Locality Name              Drug Class Death Count  \
0           2018     51590      Danville               All Drugs           8   
1           2016     51007        Amelia             Any Opioids           *   
2           2015     51115       Mathews       Rx N SS Methadone           0   
3           2018     51790      Staunton  S Opioids No Methadone           *   
4           2015     51830  Williamsburg  S Opioids No Methadone           0   

  Death Rate  
0      19.66  
1          *  
2          0  
3          *  
4          0

### Verify data is available for each county/city and year

In [11]:
vdh[vdh['Drug Class'] == 'Any Opioids'].describe()

Year of Death      Locality
count     532.000000    532.000000
mean     2016.500000  51263.992481
std         1.119086    271.113919
min      2015.000000  51001.000000
25%      2015.750000  51069.000000
50%      2016.500000  51139.000000
75%      2017.250000  51550.000000
max      2018.000000  51840.000000

There are 133 counties & cities in VA with 4 years of data for each locality, resulting in 532 data points overall.